docs/networking/cilium.md

```diff
@@ -29,7 +29,7 @@ kops create cluster \
 
 {{ kops_feature_table(kops_added_beta='1.18', kops_added_default='1.26') }}
 
-By default, Cilium will use CRDs for synchronizing agent state. This can cause performance problems on larger clusters. As of kOps 1.18, kOps can manage an etcd cluster using etcd-manager dedicated for cilium agent state sync. The [Cilium docs](https://docs.cilium.io/en/stable/gettingstarted/k8s-install-external-etcd/) contains recommendations for when this must be enabled.
+By default, Cilium will use CRDs for synchronizing agent state. This can cause performance problems on larger clusters. As of kOps 1.18, kOps can manage an etcd cluster using etcd-manager dedicated for cilium agent state sync. The [Cilium docs](https://docs.cilium.io/en/v1.12/gettingstarted/k8s-install-external-etcd/) contains recommendations for when this must be enabled.
```

Comment:
```suggestion
By default, Cilium will use CRDs for synchronizing agent state. This can cause performance problems on larger clusters. As of kOps 1.18, kOps can manage an etcd cluster using etcd-manager dedicated for cilium agent state sync. The [Cilium docs](https://docs.cilium.io/en/stable/installation/k8s-install-external-etcd/) contains recommendations for when this must be enabled.
```